src/data/nat/parity.lean

```diff
@@ -191,13 +190,10 @@ begin
     apply even_mul_succ_self }
 end
 
-lemma even_sub_one_of_prime_ne_two {p : ℕ} (hp : prime p) (hodd : p ≠ 2) : even (p - 1) :=
```

Comment:
Can you restore this one (in another file if needed)? It can definitely be useful.

Reply:
It's just `nat.odd.sub_odd (nat.prime.odd_of_ne_two _) odd_one`. Do we need a special name for this?

Reply:
I think it's very common to need this, so a lemma seems justified.

Reply:
Restored.